internal/pkg/develop/develop.go

```diff
@@ -0,0 +1,31 @@
+package develop
+
+import (
+	"github.com/merico-dev/stream/internal/pkg/develop/plugin"
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+type Action string
+
+const (
+	ActionCreatePlugin Action = "create-plugin"
+)
+
+var ActionSet = map[Action]struct{}{
```

Comment:
Is "ActionSet" needed to be exported?

Reply:
Yes, it will be use at `cmd/develop` package